README.rst

```diff
@@ -7,6 +7,20 @@ This ZenPack monitors the Linux Operating System.
 Releases
 --------
 
+.. _Version-2.2.5: http://wiki.zenoss.org/download/zenpacks/ZenPacks.zenoss.LinuxMonitor/2.2.5/ZenPacks.zenoss.LinuxMonitor-2.2.5.egg
+
+Version-2.2.5_
+   | Released on 2017/07/XXXX
+   | Compatible with Zenoss 4.2 - 5.2
+
+
+.. _Version-2.2.4: http://wiki.zenoss.org/download/zenpacks/ZenPacks.zenoss.LinuxMonitor/2.2.4/ZenPacks.zenoss.LinuxMonitor-2.2.4.egg
+
+Version-2.2.4_
```

Comment:
You can drop the 2.2.4 and 2.2.3 release sections. We only keep the latest within each feature version in this section.